src/core/server/http/http_service.ts

```diff
@@ -169,7 +169,7 @@ export class HttpService
    * @internal
    * */
   private shouldListen(config: HttpConfig) {
-    return !this.coreContext.env.isDevClusterMaster && config.autoListen;
+    return !this.coreContext.env.isDevClusterManager && config.autoListen;
   }
```

Comment:
I think we need to support both isDevClusterMaster and isDevClusterManager. Maybe this could be changed to 

```
return (!this.coreContext.env.isDevClusterManager || !this.coreContext.env.isDevClusterMaster) && config.autoListen;
```

Reply:
> Thanks @manasvinibs for starting the cleaning up. I agree with Rocky that we need to keep both `env.isDevClusterMaster ` and `env.isDevClusterManager` in env.options to not break current cx usage. Meanwhile, I think `isMaster` doesn't need to rename. Because it is a function from node which is not controlled by us and is imported from node cluster index.
> 
> ```
> import { isMaster } from 'cluster';
> ```

Thanks @kavilla and @ananzh for the feedback comments. I can update this PR to support usages of both the fields. Also, I think we should correctly update the description in the issue here https://github.com/opensearch-project/OpenSearch-Dashboards/issues/1690 to call out the changes should include deprecating the field and not to replace the existing one! 

For ```isMaster``` usages in the repo which are referenced from node module, I discussed with Rocky on the possible solutions for now 1) we can keep it as it is and may be add  a //ToDo comment 2) Add a alias where possible to restrict the deprecated nomenclature usages in OSD repo as much as we can. 
I don't have any concern in using alias for the imports here but I'm not sure if that is something we want to do every time we import which is not maintained by us. 

> I think we need to support both isDevClusterMaster and isDevClusterManager. Maybe this could be changed to
> 
> ```
> return (!this.coreContext.env.isDevClusterManager || !this.coreContext.env.isDevClusterMaster) && config.autoListen;
> ```

Shouldn't this be 
``` return !(this.coreContext.env.isDevClusterManager || this.coreContext.env.isDevClusterMaster) && config.autoListen;   ```

Reply:
yeah you are right on this and I am okay with using an alias.